projects/plugins/jetpack/modules/masterbar/profile-edit/bootstrap.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Bootstrap the WP.com User profile edit restriction.
+ *
+ * @package automattic\jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+use Automattic\Jetpack\Connection\Manager as Connection_Manager;
+
+require_once __DIR__ . '/profile-edit.php';
+require_once __DIR__ . '/class-wpcom-user-profile-fields-revert.php';
+
+/**
+ * Prevent WP.com user profile fields (first_name, last_name, display_name, description) to be updated.
+ */
+function load_the_user_profile_info_revert() {
+	global $pagenow;
+
+	new WPCOM_User_Profile_Fields_Revert( $pagenow, new Connection_Manager( 'jetpack' ) );
+}
+
+\add_action( 'init', __NAMESPACE__ . '\load_the_user_profile_info_revert' );
```

Comment:
Can we target this to the specific `load-profile.php` and `load-user-edit.php' filters? I know the `WPCOM_User_Profile_Fields_Revert` constructor is bailing when we're not in such screens, but that feels less _WordPress-y_.